genesyscloud/outbound/resource_genesyscloud_outbound_messagingcampaign.go

```diff
@@ -66,7 +66,7 @@ var (
 				Type:        schema.TypeString,
 			},
 			`content_template_id`: {
-				Description: `The content template used to formulate the message to send to the contact.`,
+				Description: `The content template used to formulate the message to send to the contact. Either message_column or content_template_id is required.`,
```

Comment:
Will this work , if either of the two attributes are not provided ? If no, please add a validate function to check either of the two params is present.

Reply:
Also, you can specify that both cannot be set together using the `ConflictsWith` array here

```golang
ConflictsWith: []string{"message_column"}
```